### PR TITLE
fix a minor bug of s_client

### PR DESCRIPTION
**Pull request by ningning-718**
CLA: trivial

-CAstore's option shoule be OPT_CASTORE, instead of OPT_CAFILE

##### Checklist

- [ ] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

File: apps/s_client.c [hunk 1]
```diff
@@ -659,7 +659,7 @@ const OPTIONS s_client_options[] = {
     {"pass", OPT_PASS, 's', "Private key file pass phrase source"},
     {"CApath", OPT_CAPATH, '/', "PEM format directory of CA's"},
     {"CAfile", OPT_CAFILE, '<', "PEM format file of CA's"},
-    {"CAstore", OPT_CAFILE, ':', "URI to store of CA's"},
+    {"CAstore", OPT_CASTORE, ':', "URI to store of CA's"},
```
Comment (t8m) [inline apps/s_client.c]: Please note that no-CAstore option has the same issue. Can you please fix it as well?

**Comment by DDvO** [comment]
Pushed using `ghmerge --nobuild 11215 @t8m @DDvO`
Thanks!

**Comment by FdaSilvaYY** [comment]
This deserves a backport to 1.1.1 branch, isn't it ?
as I can see the same issue in it.

**Comment by t8m** [comment]
There is no CASTORE support in 1.1.1.

**Pull request closed without merging**